mock/cpp_mock_scenarios/src/measurement/get_longitudinal_distance.cpp

```diff
@@ -30,10 +29,7 @@ class GetLongitudinalDistanceScenario : public cpp_mock_scenarios::CppScenarioNo
 {
 public:
   explicit GetLongitudinalDistanceScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "get_longitudinal_distance",
-      ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map", "lanelet2_map.osm",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.